test/minter-suite-minters/MinterMerkle/MinterMerkleV1.test.ts

```diff
@@ -165,6 +171,11 @@ describe("MinterMerkleV1", async function () {
     // mock ERC20 genArt721Core
     const ERC20Factory = await ethers.getContractFactory("ERC20Mock");
     this.ERC20Mock = await ERC20Factory.deploy(ethers.utils.parseEther("100"));
+
+    fakeDelegationRegistry = await smock.fake('IDelegationRegistry', {
+      address: '0x00000000000076A84feF008CDAbe6409d2FE638B'
```

Comment:
@ryley-o found a great pattern using [Smock](https://smock.readthedocs.io/en/latest/fakes.html#options) so that we can mock `checkDelegateForContract`. In the future we can prob use this to mock method calls in our unit tests whenever we have an external contract dependency.

Reply:
🙌